e2e_tests/test.ts

```diff
@@ -0,0 +1,25 @@
+import 'zx/globals';
```

Comment:
This is good start, I would make this Actor, which can be push to Apify. 
The tricky part will be to distinguish between local run and run in platform.
Localy it will be basically same as you have now and you need to add some ifs, in case you run it on platform
1. Use Actor.call instead npm run start:dev
2. Input from Actor.getInput not local file.
3. Save results to dataset maybe.